src/__test__/auth.test.js

```diff
@@ -84,6 +82,6 @@ describe('oauth', () => {
 
     await expect(() => auth({}))
       .rejects
-      .toThrowError('There was a problem exchanging a token')
+      .toThrow('There was a problem exchanging a token')
```

Comment:
`.toThrowError` is marked as deprecated in favor of `.toThrow`. I've addressed that as well.